src/main/java/com/powsybl/openloadflow/ac/outerloop/tap/GroupVoltageControlManager.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.openloadflow.ac.outerloop.tap;
+
+import com.powsybl.openloadflow.network.LfBranch;
+import com.powsybl.openloadflow.network.LfBus;
+import com.powsybl.openloadflow.network.LfNetwork;
+import com.powsybl.openloadflow.network.LfVscConverterStation;
+import com.powsybl.openloadflow.network.TransformerVoltageControl;
+import com.powsybl.openloadflow.network.VoltageControl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Didier Vidal {@literal <didier.vidal-ext at rte-france.com>}
+ */
+public class GroupVoltageControlManager {
```

Comment:
I don't understand why you need a dedicated class for that, and with `Group` inside the name: I don't see the link with group here. And all of this is not optional so for me it should be in  `TransformerVoltageControlOuterLoop`.